libs/langchain/langchain/document_loaders/parsers/msword.py

```diff
@@ -0,0 +1,34 @@
+from typing import Iterator
+
+from langchain.document_loaders.base import BaseBlobParser
+from langchain.document_loaders.blob_loaders import Blob
+from langchain.schema import Document
+
+
+class MsWordParser(BaseBlobParser):
```

Comment:
(can be refactored later) could expose an init to take a parameter (e.g., 'unstructured-doc' or 'unstructured-docx') and set the parser based on that, it'll allow users to also swap out strategies for parsing doc and docx files